pkgs/by-name/tp/tpm2-pkcs11/package.nix

```
@@ -80,12 +80,14 @@ chosenStdenv.mkDerivation (finalAttrs: {
     [
       (lib.enableFeature finalAttrs.doCheck "unit")
       (lib.enableFeature finalAttrs.doCheck "integration")
+
+      # Strangely, it uses --with-fapi=yes|no instead of a normal configure flag.
+      (lib.withFeatureAs true "fapi" (if fapiSupport then "yes" else "no"))
```

This feels a bit less convoluted, but I believe that this a matter of taste:

```suggestion
      "--with-fapi=${if fapiSupport then "yes" else "no"}"
```